autoleagueplay/replays.py

```diff
@@ -52,10 +100,21 @@ def on_modified(set_replay_id_self, event):
                 replay_path = Path(event.src_path)
                 if replay_monitor.replay_preference == ReplayPreference.CALCULATED_GG:
                     upload_to_calculated_gg(replay_path)
+                if replay_monitor.replay_preference == ReplayPreference.ANONYMIZE_REPLAY:
+                    anonymize_replay(replay_path, replay_monitor.working_dir)
+                replay_monitor.replay_id = parse_replay_id(replay_path)
+
+            def on_created(set_replay_id_self, event):
```

Comment:
Why is `self` called `set_replay_id_self`?

Reply:
https://github.com/RLBot/AutoLeaguePlay/blob/6d168257b8c7433b7cff29c3cc5acd4c076d3382/autoleagueplay/replays.py#L48

I just kept like it was before